### DISCUSSION TITLE
Approve Translation suggestions? (German)

### DISCUSSION THREAD

**Discussion by alphanimal** [Stellarium/stellarium#2011] [Translations]
Hi!

I made some corrections of outdated (as of 1996) uses of the Eszett (ß) character in the German translation.
The "Save changes" button is not available for me for reviewed translations, so I clicked "Save as suggestion" instead.

I found one instance of a wrongly used Eszett that already had a suggestion from 4 years ago that wasn't approved yet.

Is there a way you can look through my suggestions and approve them? Or should I not do any translations if I don't have the permission to save changes?

The Eszett errors were just the first thing i came across and I wanted to correct all of them at once. I'm sure I'll have more fixes coming and didn't want to continue without asking about the "suggestions" feature first.

Thanks!
Daniel

**Comment by gzotti** [comment]
There are probably 4500 further occurrences of ß. I simply prefer the consistent application of rules of the earlier 20th century over the haphazard "neue Rechtschreibung". Do you think it is worth adding "New German" as additional language?

There are still many entirely untranslated terms. In some cases I wanted to improve the original English term before translation. But every suggestion of an untranslated string will be immediately used. Approval is then my job, exclusively.

**Reply by alphanimal** [reply]
Ah thanks!

I looked through all the occurrences (just searched for 'ß' in the editor). That should have listed them all, right? I don't know if it was 4500 though... seems like a high number but it might have been that many!

I'd say it's better to use the correct spelling, rather than an old spelling that's generally not been used for 25 years.
Right now it is not consistent, because there are many instances of words that changed and are now translated in the new way. (for example Kompaß/Kompass)

I think I got all of them, so it should also be consistent now, if you approve all those suggestions. We could keep "Old German" as it's own language too, because "New German" is not that new anymore :)

Liebe Grüße,
Daniel
